app/src/main/java/com/sci/recipeandroid/feature/cart/ui/adapter/GroupedIngredientAdapter.kt

```diff
@@ -0,0 +1,82 @@
+package com.project.addtocart.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.sci.recipeandroid.R
+import com.sci.recipeandroid.feature.cart.domain.model.IngredientDetailModel
+import com.sci.recipeandroid.feature.cart.domain.model.RecipeDetailModel
+import com.sci.recipeandroid.feature.cart.ui.model.IngredientUiModel
+import com.sci.recipeandroid.feature.cart.ui.model.RecipeUiModel
+import com.sci.recipeandroid.feature.cart.ui.viewmodel.AddToCartViewModel
+
+class GroupedIngredientAdapter(
+    private val viewModel: AddToCartViewModel,
```

Comment:
view model should be use in as dependency in screen level , using in adapter make unable to reuse this adapter later